Source/WebGPU/WGSL/AST/ASTVisitor.cpp

```diff
@@ -459,8 +459,8 @@ void Visitor::visit(AST::Structure& structure)
 {
     for (auto& attribute : structure.attributes())
         checkErrorAndVisit(attribute);
-    for (auto& member : structure.members())
-        checkErrorAndVisit(member);
+    for (auto* member : structure.members())
+        checkErrorAndVisit(*member);
```

Comment:
I wrote `UniqueRefVector` to avoid this situation. If you replace `StructureMember::List` with `using List = WTF::RefVector<StructureMember>;`, you'll get the same benefit.

Reply:
Isn't RefVector a vector of WTF::Vector? That wouldn't work here. But I'm planning to use std::reference_wrapper as @litherum suggested